cfme/v2v/migrations.py

```diff
@@ -316,20 +316,41 @@ class MigrationDashboardView(BaseLoggedInPage):
     migration_plans_not_started_list = MigrationPlansList("plans-not-started-list")
     migration_plans_completed_list = MigrationPlansList("plans-complete-list")
     migration_plans_archived_list = MigrationPlansList("plans-complete-list")
-    infra_mapping_list = InfraMappingList("infra-mappings-list-view")
-    migr_dropdown = MigrationDropdown(text="Not Started Plans")
-    sort_type_dropdown = Dropdown(text="Name")
+    sort_type_dropdown = SelectorDropdown('id', 'sortTypeMenu')
     sort_direction = Text(locator=".//span[contains(@class,'sort-direction')]")
     # TODO: XPATH requested to devel (repo:miq_v2v_ui_plugin issues:415)
     progress_card = MigrationProgressBar(locator='.//div[3]/div/div[3]/div[3]/div/div')
+    not_started_plans = MigrationDashboardStatusCard(name="Not Started")
+    in_progress_plans = MigrationDashboardStatusCard(name="In Progress")
+    completed_plans = MigrationDashboardStatusCard(name="Complete")
+    archived_plans = MigrationDashboardStatusCard(name="Archived")
+    # TODO: next 3 lines are specialized only for 510z and inheritance for DashboardView59z
+    # is incorrect. Need to fix it.
+    paginator_view = View.include(v2vPaginatorPane, use_parent=True)
+    search_box = TextInput(locator=".//div[contains(@class,'input-group')]/input")
+    clear_filters = Text(".//a[text()='Clear All Filters']")
 
     @property
     def is_displayed(self):
-        return (self.navigation.currently_selected == ['Compute', 'Migration'] and
+        # TODO: Remove next line, after fix for https://github.com/ManageIQ/manageiq-v2v/issues/726
+        # has been backported to downstream 510z
+        return ((self.navigation.currently_selected == ['Compute', 'Migration'] or
+            self.navigation.currently_selected == ['Compute', 'Migration', 'Overview']) and
             (len(self.browser.elements(".//div[contains(@class,'spinner')]")) == 0) and
             (len(self.browser.elements('.//div[contains(@class,"card-pf")]')) > 0) and
             len(self.browser.elements(".//div[contains(@class,'pficon-warning-triangle-o')]")) < 1)
 
+    def switch_to(self, section):
+        """Switches to Not Started, In Progress, Complete or Archived Plans section."""
+        sections = {
+            'Not Started Plans': self.not_started_plans,
```

Comment:
TODO: when refactoring these views to be nested and conditional, let's make these view constants so callers don't have to match a specific string.

Reply:
I will add that TODO in code. And update the PR.